### PR TITLE
Adjust multiple open

### PR DESCRIPTION
**Pull request by sennett**
# [Adjust mutiple](https://app.shortcut.com/oazo-apps/story/4827/configure-multiple-for-opening-steth-aave-positions)
  
## Changes 👷‍♀️
  <Please add short list of changes>
- adds UI for adjusting multiple
- Does not include real maths yet.
  
## How to test 🧪
  <Please explain how to test your changes>
- load and get to multiple screen - it should render ok

### CODE REVIEW

File: features/earn/aave/open/services/tmpMaths.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
+import BigNumber from 'bignumber.js'
```
Comment (sennett) [inline features/earn/aave/open/services/tmpMaths.ts:1]: This is a temporary file that will get removed when the maths is worked out.

**Comment by jakubswierczek** [comment]
CleanShot 2022-09-09 at 09 57 51@2x
I think we shoud change the copy. Next step should be - `adjust risk`

**Review by jakubswierczek — DISMISSED** [review]

**Review by jakubswierczek — APPROVED** [review]

File: features/earn/aave/AaveContext.ts [hunk 2]
```diff
@@ -47,6 +51,14 @@ export function setupAaveContext({
     switchMap(({ account }) => accountBalances$(account)),
   )
 
+  const aaveReserveConfigurationData$ = observe(
+    once$,
+    connectedContext$,
+    getAaveReserveConfigurationData,
+  )
+
+  const aaveAssetPriceData$ = observe(once$, connectedContext$, getAaveAssetPriceData)
```
[the remark this answers is not in the document]
Comment (sennett) [reply features/earn/aave/AaveContext.ts:60]: It's only observed `once$`, so it's the same as a promise.

File: features/earn/aave/open/sidebars/SidebarOpenAaveVault.tsx [hunk 3]
```diff
@@ -154,6 +162,121 @@ function OpenAaveSuccessStateView({ state, send }: OpenAaveStateProps) {
...
+function SettingMultipleView({ state, send }: OpenAaveStateProps) {
+  const { t } = useTranslation()
+  function convertMultipleToColRatio(multiple: BigNumber): BigNumber {
+    return one.div(multiple.minus(one)).plus(one)
+  }
+
+  function convertColRatioToMultiple(colRatio: BigNumber): BigNumber {
+    return convertMultipleToColRatio(colRatio)
+  }
+  const marketStEthEthPrice = amountFromWei(new BigNumber('968102393798180700'), 'ETH')
```
[the remark this answers is not in the document]
Comment (sennett) [reply features/earn/aave/open/sidebars/SidebarOpenAaveVault.tsx:176]: Yes this will come from maths lib.

**Pull request merged**